.github/workflows/main.yaml

```diff
@@ -40,5 +40,5 @@ jobs:
         run: |
           export PATH="/usr/share/miniconda/bin:$PATH"
           source activate testing
-          pip install deid-data==0.0.17
+          pip install deid-data==0.0.19
```

Comment:
Do we need to pin this? I think we'd be better off always using the most up to date.

Reply:
Not from my perspective.  It was already pinned, so I just updated the version.  Committed the change.